### ISSUE TITLE
liHeight does not consider custom option content when computing li height

### ISSUE THREAD

**Issue by Mobe91** [snapappointments/bootstrap-select#2393]
Currently, liHeight creates an artificial `li` element with static content to compute its `offsetHeight`. This height is later used for virtual scrolling.
However, when a user supplies custom content via the `data-content` attribute, the current approach does not yield the correct li height.

**Comment by caseyjhol** [comment]
Bug reports must include a live demo of the problem. Per our [contributing guidelines](https://github.com/snapappointments/bootstrap-select/blob/master/CONTRIBUTING.md), please create a reduced test case (you can use our preconfigured [Plunker](https://developer.snapappointments.com/bootstrap-select/playground/)), and report back with:

- your link
- Bootstrap version
- bootstrap-select version
- specific browser and OS details.

If you're not using the latest version of bootstrap-select, please try that as well.

**Comment by Mobe91** [comment]
Here is a reproducer: https://plnkr.co/edit/MAlf9YRQoz948SPM

Bootstrap version: 4.1.0
Bootstrap-select version: v1.13.15
Tested in Firefox on Windows 10

I have changed the first select to use custom html option content + virtual scroll. Just scroll to the very end of the option list and you will see blank content caused by wrong li height calculation. The reproducer is fixed by applying the changes from this PR.
